code/modules/mob/living/emote.dm

```diff
@@ -182,6 +189,22 @@
 	emote_type = EMOTE_AUDIBLE
 	stat_allowed = HARD_CRIT
 
+/datum/emote/living/gasp/get_sound(mob/living/user)
+	if(iscarbon(user))
+		if(user.gender == MALE)
+			return pick('sound/emotes/male/gasp_m1.ogg',
```

Comment:
A lot of these could really do with having early returns, but considering how many there are that's probably out of scope of this PR.